tests/functional/g/generated_members.txt

```diff
@@ -0,0 +1 @@
+no-member:13:6::Instance of 'Klass' has no 'spam' member:INFERENCE
```

Comment:
```suggestion
no-member:13:15::Instance of 'Klass' has no 'spam' member:INFERENCE
```
If you're feeling brave the column can probably be placed at the `s` of spam, but this is a different issue than the one you're trying to fix :)